debian/build-debs

```diff
@@ -41,8 +41,8 @@ cd ${unpacked_src_dir}
 # *.so exclusion removed.
 excludes='-I*/*~ -I,,* -I.[#~]* -I.git -I.gitattributes -I.gitignore -I.gitmodules -I.gitreview'
 
-debuild -i="" $excludes -us -uc -b
-debuild -i="" $excludes -us -uc -S
+debuild --no-lintian -i="" $excludes -us -uc -b
```

Comment:
(Well, at the moment it's really just generating a ton of errors in our build output, and our Debian packages are parsecs away from meeting Debian policy!)